changelog.txt

```diff
@@ -1,10 +1,9 @@
-Version 3.10.0, February 1, 2020
-
-**User-visible changes:**
+Version 3.9.1, January 12, 2020
 
 **Implementation details:**
 
-Moved methods on `TreePath`s from class 'TreeUtils` to new class `TreePathUtil`.
+Copied methods on `TreePath`s from class 'TreeUtils` to new class `TreePathUtil`.
+(The methods in TreePath will be deleted in the next minor release.)
```

Comment:
Nit: I would change "next minor release" to "next release", to avoid using jargon that may be confusing to those reading the changelog.